### PR TITLE
Home Page

### LINKED ISSUE #1: [Website] Home Page

### LINKED ISSUE #5: [Website] Header / Footer Page

### LINKED ISSUE #6: [Website] Images

### PR DESCRIPTION
**Pull request by redpandarevolt**
Index.html 
Images - all copyright free
footer - copy and paste onto other pages
header - copy and paste onto other pages
media queries at 4 break points
Links to external sites are working
Link to internal pages need to be linked to correct .html page from other students.

### CODE REVIEW

**Review by mduw — CHANGES_REQUESTED** [review]
looks great to me overall. 
Suggest putting .jpg files into img folder.

File: index.html
Comment (isabelgiang) [inline index.html]: Should these `width` and `height` attributes be moved out into the CSS file?

**Comment by isabelgiang** [comment]
Looks great to me 👍

File: index.html
Comment (redpandarevolt) [reply index.html]: CSS can be either in line on html page or in style sheet depending on how you want it to function.  The measurements were put directly on the image reference then added a width of 100% in CSS so they adjust with each devices width.   This is part of using the view-port option for device resizing.  One of many ways to skin the cat.  I'll look into it and see if there needs to be a modification.

**Comment by redpandarevolt** [comment]
> looks great to me overall.
> Suggest putting .jpg files into img folder.

Mitchel that is a good idea for any image that is not linked.  If the image is a link will it cause he URL to be longer? As in it will turn into http://t4g-website/image/bike.jpg   vs  http://t4g-website/bike.jpg.  I'm not sure what is considered best practice on this.  

If it doesn't matter then I'm fine dumping all pictures into an /image folder. 
Good call!

**Comment by redpandarevolt** [comment]
I also need to add commenting so future users have a good working idea of the moving parts and how to work with the site.

**Comment by redpandarevolt** [comment]
> looks great to me overall.
> Suggest putting .jpg files into img folder.

done!

**Review by isabelgiang — APPROVED** [review]

**Comment by isabelgiang** [comment]
@mduw can you approve the changes that @redpandarevolt has made on this PR? since you requested changes previously, merging is blocked until you approve those changes

**Comment by isabelgiang** [comment]
> looks great to me overall.
> Suggest putting .jpg files into img folder.

this review specifically

**Review by mduw — APPROVED** [review]

**Pull request merged**
